File: 151004/Bosko/lab1/lab1/DTO/UserRequestTo.cs
using lab1.DTO.Interface;

namespace lab1.DTO
{
    public class UserRequestTo(int Id, string? Login, string? Password, string? Firstname, string? Lastname) : IRequestTo
    {
        public int Id { get; set; } = Id;

        public string? Login { get; set; } = Login;

        public string? Password { get; set; } = Password;

        public string? Firstname { get; set; } = Firstname;

        public string? Lastname { get; set; } = Lastname;
    }
}

File: 151001/Yankova/RV1/src/main/java/by/bsuir/nastassiayankova/Dto/impl/UserRequestTo.java
package by.bsuir.nastassiayankova.Dto.impl;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class UserRequestTo {
    private long id;
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}
